web/client/epics/identify.js

```diff
@@ -223,7 +224,7 @@ export default {
         })
             .switchMap(({point, layer}) => {
                 const projection = projectionSelector(store.getState());
-                return Rx.Observable.of(featureInfoClick(updatePointWithGeometricFilter(point, projection), layer))
+                return Rx.Observable.of(featureInfoClick(updatePointWithGeometricFilter(point, projection), layer), cancelSelectedItem())
```

Comment:
can you add a unit test here to check that this action is dispatched as feature info click?

Reply:
Hi Matteo, 

Isn't this test case below fulfill what you describe? 

https://github.com/geosolutions-it/MapStore2/blob/59715dfd851d713c5b9f3846f327eaa61c839ba2/web/client/epics/__tests__/identify-test.js#L558

I thought that this case was checking `if this action is dispatched as a feature info click`?

Please guide me if I misunderstood what you requested

Reply:
if that test is the one affetecd by your changes then 
maybe you need to add a case check for the new action dispatched by increasing the number of actions expected.
if you need more help we can have a screenshare session

Reply:
@MV88 I have updated the tests as you suggested